src/app/layout/components/app-layout/app-layout.component.html

```diff
@@ -52,8 +52,8 @@
         fxFlex.gt-md="90%"
         fxFlex.gt-lg="1200px"
       >
-        <router-outlet *ngIf="!unapprovedUser"></router-outlet>
-        <div *ngIf="unapprovedUser">
+        <router-outlet *ngIf="!unapprovedUser || onHomePage"></router-outlet>
+        <div *ngIf="unapprovedUser && !onHomePage">
```

Comment:
create a method and use these conditions ex. : isRouterOutletDisplayed

Reply:
Done.